plugin/telescope.vim

```diff
@@ -27,6 +27,18 @@ highlight default link TelescopePromptPrefix Identifier
 highlight default link TelescopePreviewLine Visual
 highlight default link TelescopePreviewMatch Search
 
+highlight default link TelescopePreviewDirectory Directory
+highlight default link TelescopePreviewNormal Normal
+highlight default link TelescopePreviewRead Constant
+highlight default link TelescopePreviewWrite Statement
+highlight default link TelescopePreviewExecute String
+highlight default link TelescopePreviewNon NonText
+highlight default link TelescopePreviewSticky Keyword
+highlight default link TelescopePreviewSize String
+highlight default link TelescopePreviewUser Constant
+highlight default link TelescopePreviewGroup Constant
+highlight default link TelescopePreviewDate Directory
```

Comment:
I am a noob in hl groups. Thats the best i could do :)

Reply:
`TelescopePreviewNon` is a bit ambiguous, I think. What does it do? (reading from the code, I think it highlights `-`)

Reply:
Yeah. Highlights `-` so if a permission is not set.

But i also realized i forgot to add those filetypes so if we have a `b` (block device) we error:
```
local type_tbl = { [1]  = 'p', [2]  = 'c', [4]  = 'd', [6]  = 'b', [10] = '-', [12] = 'l', [14] = 's' }
```

Gotta fix that tomorrow :)
If you have a better name for `TelescopePreviewNon` i would be very happy and thankful :)

Reply:
Btw just putting this here. We kinda never get a symbolic link. `uv` gives me the same octal for a link as for a file. :|

`Hypen` sounds good :) Thanks elianiva

Reply:
> Btw just putting this here. We kinda never get a symbolic link. uv gives me the same octal for a link as for a file. :|

Yeah, it looks a bit different from doing `ls -l`, Telescope shows `drwxr-xr-x` while `ls -l` shows `lrwxrwxrwx`.  But it doesn't really matter to me personally :laughing:

Reply:
I think whats happening is that `uv.fs_stat` actually resolves it. Even the inode is the same which it isn't. Or do i resolve it? No i don't think i need to investigate :rofl:

Reply:
But doesn't change anything. Doing this with `a` being file and `b` a symbolic link of `a` gives me the same inode/mode
```lua
local P = function(...) print(vim.inspect(...)) end
P(vim.loop.fs_stat('a'))
P(vim.loop.fs_stat('b'))
```

Reply:
`lstat` works. So doing `vim.loop.fs_lstat`